sympy/stats/tests/test_continuous_rv.py

```diff
@@ -1031,7 +1031,7 @@ def test_density_unevaluated():
     Y = Normal('Y', 0, 2)
     assert isinstance(density(X+Y, evaluate=False)(z), Integral)
 
-
+@XFAIL # sample function now returns array
```

Comment:
I also don't want to break the API but one change this PR introduces is that now every `_sample_python` function will return an array. Which will make `assert isinstance(nd.sample(), float) or nd.sample().is_Number` statement `False`.
I think that we should be returning an array since most of the time, we require samples > 1.

Reply:
We are not going to change the current API. One possible solution is to pass the shape of the array to generate, so that `()` corresponds to scalars. `shape=()` could be made the default value.